json/src/lib.rs

```diff
@@ -1140,6 +1140,39 @@ pub enum PubKeyOrAddress<'a> {
     PubKey(&'a PublicKey),
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(untagged)]
+pub enum ScanTxoutRequest {
+    Single(String),
+    Extended {
+        dest: String,
+        range: u64,
```

Comment:
`range` can also be a tuple of `(start_index,end_index)`.

Reply:
I guess I'll have to impl a custom serializer for that as it has either to be a number _or_ a array with two numbers (super annoying). Alternatively we could require a lower limit.

Reply:
I think that requiring a lower limit is reasonable, adding an explicit `0` start range is not much of an hassle and allows to keep things simpler.

Reply:
Fixed, made it a tuple. Maybe it should actually be `(u23, u32)` though? Afaik key derivation can only go up to `2^31-1` for each hardened and unhardened keys according to bip32. So u64 is a bit too big, but u32 too and I don't really want to get into using `ChildNumber` here.